testing/frr/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0
+pkgrel=0
+pkgdesc="Free Range Routing is a fork of quagga"
+url="https://frrouting.org/"
+arch="x86_64"
+license="GPL-2.0"
+depends="json-c c-ares ipsec-tools iproute2 python py-ipaddr bash"
```

Comment:
Add the following instead of using `_user=frr`
```
pkgsusers="frr"
pkggroups="frr"
```

Reply:
Thanks, will do...